dev-packages/browser-integration-tests/suites/tracing/browserTracingIntegration/long-tasks-and-animation-frame-enabled/test.ts

```diff
@@ -102,6 +100,7 @@ sentryTest(
         data: {
           'browser.script.invoker': 'BUTTON#clickme.onclick',
           'browser.script.invoker_type': 'event-listener',
+          'code.filepath': 'https://example.com/path/to/script.js',
```

Comment:
this is new, but seems correct to me!

Reply:
Probably came from the browser version bump 🤔